repositories.yaml

```diff
@@ -180,6 +180,25 @@ applications:
       - v1_name: firefox-desktop
         app_id: firefox.desktop
 
+  - app_name: firefox_desktop_background_updater
+    canonical_app_name: Firefox for Desktop Background Updater
+    app_description: >-
+      The desktop version of Firefox's background update tasks. This is
+      considered a seperate application because it has its own client
+      id and sends its own version of the standard pings.
+    url: https://github.com/mozilla/gecko-dev
+    notification_emails:
+      - chutten@mozilla.com
+    metrics_files:
+      - toolkit/mozapps/update/metrics.yaml
+    ping_files:
+      - toolkit/mozapps/update/pings.yaml
+    dependencies:
+      - glean-core
+    channels:
+      - v1_name: firefox-desktop-background-updater
+        app_id: firefox.desktop.backgroundupdate
```

Comment:
[I now prefer `firefox.desktop.background.update`](https://mozilla.slack.com/archives/C4D5ZA91B/p1619033249269600?thread_ts=1619029486.253700&cid=C4D5ZA91B) so that we have a separator between `background` and `update` which matches the name of the ping (`background_update`).